ajax/libs/folktale/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "folktale",
+  "filename": "umd/folktale.min.js",
```

Comment:
Should `umd/` be removed?

And the file structure need to move from A to B?
- A
```
ajax/libs/folktale/2.0.1/umd/folktale.js
ajax/libs/folktale/2.0.1/umd/folktale.min.js
```
- B
```
ajax/libs/folktale/2.0.1/folktale.js
ajax/libs/folktale/2.0.1/folktale.min.js
```

Thanks.